modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -1416,7 +1416,7 @@ private CompletableFuture<Void> changePeersOnRebalance(
             Set<Assignment> pendingAssignments,
             long revision
     ) {
-        return replicaMgr.replica(replicaGrpId)
+        replicaMgr.replica(replicaGrpId)
```

Comment:
Please leave a comment that we intentionally ignore the returned future

Reply:
Added.